docs/bcfusion/validators/staking.md

```diff
@@ -0,0 +1,133 @@
+- **New vote address**: The new vote address for participating fast finality.
+
+These transactions will update the validator information on the BNB smart chain, and the changes will take
+effect immediately. However, the new commission rate will only apply to the rewards earned after the transaction, and
+the previous rewards will be distributed according to the previous commission rate.
+
+## Delegator Operations
+
+Delegators are the BNB holders who stake their BNB to a validator of their choice, and share the rewards and the risks
+with the validator. Delegators can choose any validator from the active or the standby validator set, and they can
+switch between validators at any time. Delegators can also undelegate their BNB from a validator, and claim their
+rewards at any time.
+
+### Delegate
+
+To delegate BNB to a validator, a BNB holder needs to send a `Delegate` transaction to the BNB smart chain, specifying
+the following information:
+
+- **Operator address**: The BEP20 address of the validator, which will receive the BNB from the delegator.
+- **Delegate Voting Power**: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the validator for governance.
+
+The `Delegate` transaction will deduct the amount of BNB from the delegator address and issue the corresponding staking
+credit to the validator. The delegator will then share the rewards and the risks with the validator, according to the
+commission rate and the slashing rate of the validator.
+
+### Redelegate
+
+To redelegate BNB from one validator to another, a delegator needs to send a `Redelegate` transaction to the BNB smart
+chain, specifying the following information:
+
+- **Source operator address**: The BEP20 address of the source validator, which will send the BNB to the destination
+  validator.
+- **Destination operator address**: The BEP20 address of the destination validator, which will receive the BNB from the
+  source validator.
+- **Amount**: The amount of BNB that the delegator wants to redelegate from the source validator to the destination
+  validator.
+- **Delegate Voting Power**: The flag to indicate whether the delegator would like to delegate his/her voting power
+  to the destination validator for governance.
+
+The `Redelegate` transaction will deduct the amount of staking credit from the source validator and issue the
+corresponding staking credit to the destination validator. The delegator will then share the rewards and the risks with
+the destination validator, according to the commission rate and the slashing rate of the destination validator.
+The `Redelegate` transaction does not incur the unbonding period, but it will incur the redelegation fee,
+which is designed to prevent delegators from frequently switching between validators to chase
+the highest rewards or avoid the highest risks.
+
+### Undelegate
```

Comment:
can we explain the core logic about how user earn reward from staking..
I mean how to turn `share` to real `BNB` reward, the key formula?

Reply:
added in delegation part